static/styles/settings.css

```diff
@@ -1303,6 +1303,9 @@ input[type="checkbox"] {
                 border-bottom: 1px solid transparent;
             }
 
+            &:focus {
+                outline: 1px solid hsl(0, 100%, 50%);
```

Comment:
Red? I’d rather not add yet another inconsistent focus ring color here.

Reply:
Yeah, I was confused too about what to do over here because it needs to be visible for both the day and night mode.
Can you suggest what might be better?

Reply:
In #12597 I spent a while trying to match the native focus rings in Firefox and Chrome and came up with

```css
            &:focus {
                outline: 1px dotted;
                outline: -webkit-focus-ring-color auto 5px;
            }
```

Reply:
Okay, so i will change this..